pages/docs/tutorials/message-validation.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Message validation
+description: In this tutorial, you'll learn how to validate AsyncAPI messages (events).
+
+weight: 130
+---
+
+# Introduction
+In this tutorial, you'll learn how to validate messages (events) that are sent to your AsyncAPI application.
+
+# Background context
+Message validation can be performed at both the producer and consumer levels. Message validation requires the participation of the producer, consumer, and broker. We will learn how to validate messages at the consumer level by discarding invalid messages based on the parameters provided.
+
+You will be using the [Eclipse Mosquitto](https://mosquitto.org/) broker. The MQTT protocol provides a lightweight method of messaging using a publish/subscribe model. You will also use an MQTT client that runs an MQTT library and connects to an MQTT broker over a network. Here publishers and subscribers are MQTT clients. The publisher and subscriber labels refer to whether the client is publishing or subscribing to receive messages.
+
+
+In the previous tutorial, you generated your application using the [AsyncAPI Generator](https://github.com/asyncapi/generator) Node.js template. Now you will be validating the messages (events) which you will be sending to your application using a Mosquitto broker and MQTT client.
+
+
+#  Validate messages
+In this step, we will send a message to our application using an MQTT broker and check the errors logged when you accidentally send an invalid message.
+
+1. Start your generated application.
+
+<CodeBlock language="bash">
+{`npm start`}
+</CodeBlock>
+
+2. Now let's try to send a message:
+
+<CodeBlock language="bash">
+  {`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": "3", "sentAt": "2017-06-07T12:34:32.000Z"}'`}
+  </CodeBlock>
+  
+ Go back to the previous terminal and check if your application logged the streetlight condition you just sent, with errors related to the invalid message.

```

Comment:
```suggestion
 Go back to the previous terminal and check if your application logged the streetlight condition you just sent, with errors related to the invalid message. You should see something like this displayed in the terminal:
```

and please add code snippet that shows example validation message

Reply:
OKay so I tried doing it in killer coda and it didn't show me any errors when I changed number to string but it should right?

Reply:
hmmm, I just generated code locally and works good:

```bash
SUB  Subscribed to light/measured
Streetlights API Simplified 1.0.0 is ready! 

🔗  MQTT adapter is connected!
← light/measured was received:
{ id: 1, lumens: '3', sentAt: '2017-06-07T12:34:32.000Z' }
❗  Message Rejected. data.lumens should be integer
```

I recommend playing with it on local, especially as tutorial author